examples/hello-pt/app/custom/cifar10trainer.py

```diff
@@ -139,6 +137,16 @@ def execute(self, task_name: str, shareable: Shareable, fl_ctx: FLContext, abort
             self.log_exception(fl_ctx, f"Exception in simple trainer: {e}.")
             return make_reply(ReturnCode.EXECUTION_EXCEPTION)
 
+    def _get_model_weights(self) -> Shareable:
+        # Get the new state dict and send as weights
+        new_weights = self.model.state_dict()
+        new_weights = {k: v.cpu().numpy() for k, v in new_weights.items()}
```

Comment:
I prefer rename `new_weights` to `weights`

Reply:
Good point. Renamed.